.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+    ecmaFeatures: { jsx: true },
+  },
+  settings: {
+    react: { version: 'detect' },
+  },
+  extends: [
+    'plugin:react/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'prettier/@typescript-eslint',
+    'plugin:prettier/recommended',
+  ],
+  ignorePatterns: ['src/assets/**/*.json'],
```

Comment:
Will this prevent VSCode from trying to auto-format huge JSON files? This was the purpose of the prettierignore. Do you think you could give it a shot by saving the test.json map file?

Reply:
Hm. It doesn't reformat it. VS Code still attempts to format after it waits a second though. Is that expected?